backend/src/test/java/org/capstone/my_supplier/customer/OrderIntegrationTest.java

```diff
@@ -53,4 +53,48 @@ void addOrder() throws Exception {
         Assertions.assertEquals("""
                 {"orderId":"<ID>","products":[{"productId":"22","productName":"PName","itemNumber":"225588","description":"ddd","category":"OBST","quantity":"2","measurementUnit":"BUND"},{"productId":"23","productName":"PName2","itemNumber":"225599","description":"ddd","category":"KRAEUTER","quantity":"2","measurementUnit":"BUND"}]}""".replaceFirst("<ID>", actualId), content);
```

Comment:
hier vielleicht aus formatieren das man besser lesen kann. kein muss

Reply:
Hey, hier hat der Compiler eine andere Variante nicht angenommen. Liegt wohl an Objekte in einer Array in einem Json-Objekt...